index.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
```

Comment:
When it comes to JS, better to use 2 spaces as a tab. Refer Airbnb JS Style guide. We are using that guide inside 99X JS projects.